mirgecom/inviscid.py

```diff
@@ -84,30 +83,102 @@ def inviscid_flux(state):
                           momentum=mom_flux, species_mass=species_mass_flux)
 
 
-def inviscid_facial_flux(discr, state_tpair, local=False):
-    r"""Return the flux across a face given the solution on both sides *q_tpair*.
+def inviscid_flux_rusanov(state_pair, gas_model, normal, **kwargs):
```

Comment:
Why the naming inconsistency of `_lfr` vs `_rusanov`?

Reply:
The intent is that the underlying LFR numerical flux can be switched from Rusanov proper to general Lax-Friedrichs by the choice of the penalty factor "lambda".   The Rusanov-specific one chooses the max local wavespeed.